### PR TITLE
Update version resource-graph

### PR DESCRIPTION
**Pull request by OlhaTkachenko**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/index.json [hunk 1]
```diff
@@ -1893,8 +1893,8 @@
         ],
         "resource-graph": [
             {
-                "downloadUrl": "https://files.pythonhosted.org/packages/b3/a8/6862d447037fa699364c081d90ef0d39a222e93ad0b93d30711618859ee2/resource_graph-0.1.8-py2.py3-none-any.whl",
-                "filename": "resource_graph-0.1.8-py2.py3-none-any.whl",
+                "downloadUrl": "https://files.pythonhosted.org/packages/71/fc/de190502d85f724e62f714156429dca7f10b762ebe320f46219ce78e9786/resource_graph-0.1.9-py2.py3-none-any.whl",
+                "filename": "resource_graph-0.1.9-py2.py3-none-any.whl",
                 "metadata": {
                     "azext.isPreview": true,
                     "azext.minCliCoreVersion": "2.0.45",
```
Comment (demyanenko) [inline src/index.json]: Do we need maxCliCoreVersion as well?

File: src/index.json [hunk 2]
```diff
@@ -1925,7 +1925,7 @@
                     "metadata_version": "2.0",
                     "name": "resource-graph",
                     "summary": "Support for querying Azure resources with Resource Graph.",
-                    "version": "0.1.8"
+                    "version": "0.1.9"
                 },
                 "sha256Digest": "f95896b07f9a16d47307e4f6b38ffc8ba9f335f90a8661acfa2b980599d14b32"
```
Comment (demyanenko) [inline src/index.json]: Digest also needs an update

Comment (OlhaTkachenko) [reply src/index.json]: Good point! Thank you

File: src/index.json (hunk 1 above)
Comment (OlhaTkachenko) [reply src/index.json]: Added

**Comment by demyanenko** [comment]
Looks good to me. Thanks @OlhaTkachenko!

**Pull request merged**